### PR TITLE
zuul.yaml: enable debian packaging

### PR DESCRIPTION
**Pull request by fredericlepied**
Depends-On: https://github.com/wazo-platform/sf-jobs/pull/23

### CODE REVIEW

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

File: debian/rules [hunk 1]
```diff
@@ -6,3 +6,6 @@ export PYBUILD_DISABLE=test
 
 %:
 	dh $@ --with python3,systemd --buildsystem=pybuild
+
+override_dh_installinit:
+	dh_installinit --noscripts
```
Comment (sileht) [inline debian/rules]: What is it for ?

Comment (sileht) [reply debian/rules]: If I remember correctly it disable `systemctl enable --now wazo-auth', no ?

Comment (fredericlepied) [reply debian/rules]: to remove a lintian error regarding initscripts

Comment (fredericlepied) [reply debian/rules]: > If I remember correctly it disable `systemctl enable --now wazo-auth', no ?

no it's only for old form init file. systemd is stil enabled.

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
regate

**Pull request merged**